packages/realm-web/src/FunctionsFactory.ts

```diff
@@ -180,4 +182,34 @@ export class FunctionsFactory {
             body,
         });
     }
+
+    /**
+     * Call a remote function by it's name.
+     *
+     * @param name Name of the remote function.
+     * @param args Arguments to pass to the remote function.
+     * @returns A promise of the value returned when executing the remote function.
+     */
+    public callFunctionStreaming(
```

Comment:
I'm guessing this only works if the function supports streaming? If so, maybe rename to `callStreamableFunction`?

Streaming functions are then not supported through the proxied function calls, correct? We probably need to document that somewhere.

Reply:
It's mainly here for the watch support.
I'm unsure but believe that it's not possible to change the response content-type of a MongoDB Realm Function, it's always `application/json`, which would be a prerequisite for custom streaming functions defined by the users of MongoDB Realm.

Reply:
Ok, I'm just worried users will expect `callFunctionStreaming` to be a streaming implementation of `callFunction`, usable in the same way. And this would be exposed on `user.functions`, right?

Reply:
Right ..